library/demo/stock_quant.xml

```diff
@@ -0,0 +1,109 @@
+  <record id="stock_quant_9" model="stock.quant">
+    <field name="product_id" ref="product_product_16"/>
+    <field name="inventory_quantity_auto_apply">1.0</field>
+  </record>
+  <record id="stock_quant_10" model="stock.quant">
+    <field name="product_id" ref="product_product_16"/>
+    <field name="inventory_quantity_auto_apply">-1.0</field>
+  </record>
+  <record id="stock_quant_11" model="stock.quant">
+    <field name="product_id" ref="product_product_17"/>
+    <field name="inventory_quantity_auto_apply">1.0</field>
+  </record>
+  <record id="stock_quant_12" model="stock.quant">
+    <field name="product_id" ref="product_product_17"/>
+    <field name="inventory_quantity_auto_apply">-1.0</field>
+  </record>
+  <record id="stock_quant_13" model="stock.quant">
+    <field name="product_id" ref="product_product_18"/>
+    <field name="inventory_quantity_auto_apply">1.0</field>
+  </record>
+  <record id="stock_quant_14" model="stock.quant">
+    <field name="product_id" ref="product_product_18"/>
+    <field name="inventory_quantity_auto_apply">-1.0</field>
+  </record>
+  <record id="stock_quant_15" model="stock.quant">
+    <field name="product_id" ref="product_product_19"/>
+    <field name="inventory_quantity_auto_apply">1.0</field>
+  </record>
+  <record id="stock_quant_16" model="stock.quant">
+    <field name="product_id" ref="product_product_19"/>
+    <field name="inventory_quantity_auto_apply">-1.0</field>
+  </record>
+  <record id="stock_quant_17" model="stock.quant">
+    <field name="product_id" ref="product_product_20"/>
+    <field name="inventory_quantity_auto_apply">1.0</field>
+  </record>
+  <record id="stock_quant_18" model="stock.quant">
+    <field name="product_id" ref="product_product_20"/>
+    <field name="inventory_quantity_auto_apply">-1.0</field>
+  </record>
+  <record id="stock_quant_19" model="stock.quant">
+    <field name="product_id" ref="product_product_21"/>
+    <field name="inventory_quantity_auto_apply">1.0</field>
+  </record>
+  <record id="stock_quant_20" model="stock.quant">
+    <field name="product_id" ref="product_product_21"/>
+    <field name="inventory_quantity_auto_apply">-1.0</field>
+  </record>
+  <record id="stock_quant_21" model="stock.quant">
+    <field name="product_id" ref="product_product_9"/>
+    <field name="location_id" ref="stock_location_17"/>
+    <field name="inventory_quantity_auto_apply">1.0</field>
+  </record>
+  <record id="stock_quant_22" model="stock.quant">
+    <field name="product_id" ref="product_product_10"/>
+    <field name="location_id" ref="stock_location_17"/>
+    <field name="inventory_quantity_auto_apply">1.0</field>
+  </record>
+  <record id="stock_quant_23" model="stock.quant">
+    <field name="product_id" ref="product_product_14"/>
+    <field name="location_id" ref="stock_location_17"/>
+    <field name="inventory_quantity_auto_apply">1.0</field>
+  </record>
+
+  <function name="action_apply_inventory" model="stock.quant" eval="[[
```

Comment:
with `inventory_quantity_auto_apply`, you shouldn't need to call `action_apply_inventory`